go/vt/mysqlctl/mysqld.go

```diff
@@ -793,7 +794,14 @@ func (mysqld *Mysqld) initConfig(cnf *Mycnf, outFile string) error {
 }
 
 func (mysqld *Mysqld) getMycnfTemplate() string {
-
+	if *mycnfTemplateFile != "" {
+		data, err := ioutil.ReadFile(*mycnfTemplateFile)
+		if err != nil {
+			log.Warningf("template file could not be read, using default template instead: %v", *mycnfTemplateFile) // continue to default
```

Comment:
If a file was specified but could not be read for some reason I think fatal would be reasonable/expected so that the error can be diagnosed. Receiving a default mysql config is unlikely to work if you want/expect a specific one at best and could, at worst, be insecure or dangerous

Reply:
That makes sense to me too. I've changed it to fatal.